### PR TITLE
Use lodash specific import + peerDep

### PR DESCRIPTION
**Pull request by ziir**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +23,7 @@
     "babel-preset-es2015": "^6.0.15",
     "babel-preset-stage-0": "^6.0.15",
     "chai": "^3.4.1",
+    "lodash": "^4.11.1",
```
Comment (pascalduez) [inline package.json]: Missing addition as peerDependency as well.

**Comment by pascalduez** [comment]
Side note on lodash imports, since webpack 2+ we could start using `lodash-es` to benefit from tree-shaking.
https://github.com/lodash/lodash/tree/es  
https://www.npmjs.com/package/lodash-es

File: package.json (hunk 1 above)
Comment (ziir) [reply package.json]: Done

**Comment by ziir** [comment]
@pascalduez said:
> Side note on lodash imports, since webpack 2+ we could start using lodash-es to benefit from tree-shaking.

Well, as the whole purpose of this PR is to manually "tree-shake", we actually would not benefit from webpack's tree-shaking.

One thing we would benefit from though, is de-duplication within our dependency tree between `lodash` and `lodash-es`, between 0 - 20kb un-gzipped.
But that would require updating a whole lot more of internal dependencies..

**Pull request merged**